content/slice/language-guide/class-types.md

```diff
@@ -253,21 +269,32 @@ class FrontBumper : CarPart {
 ```csharp
 public partial class RearBumper : CarPart
```

Comment:
Also not from your PR, but the Slice definition is named `FrontBumper`, yet this C# class is named `RearBumper`.
Maybe give a quick `rg` to see if we make this mix-up anywhere else in this file?

Reply:
Fixed. FrontBumper is only for the Slicing discussion.